geowebcache/nestedstorage/src/main/java/org/geowebcache/nested/NestedBlobStoreConfig.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * @author Stuart Adam, Ordnance Survey, Copyright 2017
```

Comment:
@jodygarnett I am getting people to look into that for me.  I was under the impression that it was not mandatory for GWC but preferable if it could be done so hopefully I will get news on that soon.